libnymea/devices/device.cpp

```diff
@@ -366,10 +361,16 @@ void Device::setParentId(const DeviceId &parentId)
     m_parentId = parentId;
 }
 
-/*! Returns true, if setup of this Device is already completed. */
+/*! Returns true, if setup of this Device is already completed. This method is deprecated, use setupStatus() instead. */
```

Comment:
https://doc.qt.io/qt-5/16-qdoc-commands-status.html#obsolete
Maybe use the qdoc function for depreacting methods.

Reply:
Good point, yes. In this case I'll be dropping in the other branch that renames all the things...